cmsimple/languages/en.php

```diff
@@ -103,6 +103,7 @@
 
 $tx['help']['images_maxsize']="Maximum size of uploaded images in Byte. This must neither exceed the limit set for upload_max_filesize nor post_max_size in the PHP configuration.";
 $tx['help']['language_default']="The primary language of your site";
+$tx['help']['language_menu']="alternative language menu";
```

Comment:
```suggestion
$tx['help']['language_menu']="Alternative language menu";
```
```suggestion
$tx['help']['language_menu']="alternative language menu";
```

Reply:
I don't understand. Am I missing something? Somehow, some things appear twice here.

Reply:
I've updated the comment (something was/is wrong with GH UI). It's just about capitalizing the first letter (`a` → `A`).